### PR TITLE
Bug/ch83448/fix map backups with ghost tables

### PR DESCRIPTION
**Pull request by amiedes**
Closes https://app.clubhouse.io/cartoteam/story/83448/fix-map-backups-with-ghost-tables

Fixes maps backups when deleting user tables via Ghost Tables

### CODE REVIEW

File: app/helpers/logger_helper.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def rails_log(level, params)
       if value.is_a?(Exception)
         parsed_params[:exception] = { class: value.class.name, message: value.message, backtrace_hint: value.backtrace&.take(5) }
         parsed_params[:message] = value.message if params[:message].blank?
-      elsif value.is_a?(User) || value.is_a?(Carto::User)
+      elsif value.is_a?(::User) || value.is_a?(::Carto::User)
```
Comment (rafatower) [inline app/helpers/logger_helper.rb:38]: minor detail: you should also apply the same to the `Organization` below

**Review by rafatower — APPROVED** [review]
:+1: :clap: :clap: 

In the end It was different from what I had in mind (partly because of my bad memory).

**Review by rafatower — APPROVED** [review]

File: app/helpers/logger_helper.rb (hunk 1 above)
Comment (amiedes) [reply app/helpers/logger_helper.rb:38]: Thanks! I did not spot this 😉

**Comment by amiedes** [comment]
Acceptance ✅ : I dropped the `cartodb_id` column of a dataset in staging and I was able to find and restore the backup.

**Pull request merged**